prov/efa/src/efa_hmem.c

```diff
@@ -91,8 +117,7 @@ static int efa_hmem_info_update_cuda(struct efa_hmem_info *cuda_info, struct efa
 	int ret;
 
 	if (!ofi_hmem_is_initialized(FI_HMEM_CUDA)) {
-		EFA_INFO(FI_LOG_DOMAIN,
-		         "FI_HMEM_CUDA is not initialized\n");
+		EFA_INFO(FI_LOG_DOMAIN, "FI_HMEM_CUDA is not initialized\n");
```

Comment:
check the length of the line. I remember it was split to 2 lines to not break the 80 characters limit. Similar comment for other changes.

Reply:
This particular line is 79 columns wide (with tabs == 8 spaces). There are many other untouched lines in this file well over 80 columns wide.

```bash
$ sed 's/\t/        /g' prov/efa/src/efa_hmem.c | egrep '.{80,}' | wc -l
      49
```